pkg/operation/botanist/component/vpnseedserver/vpn_seed_server.go

```diff
@@ -0,0 +1,645 @@
+				deployment.Spec.Template.Spec.Containers[0].Env,
+				corev1.EnvVar{Name: "NODE_NETWORK", Value: *v.nodeNetwork},
+			)
+		}
+		return nil
+	}); err != nil {
+		return err
+	}
+
+	if _, err := controllerutil.CreateOrUpdate(ctx, v.client, configMap, func() error {
+		configMap.Labels = map[string]string{
+			v1beta1constants.GardenRole:     v1beta1constants.GardenRoleSystemComponent,
+			managedresources.LabelKeyOrigin: managedresources.LabelValueGardener,
+		}
+		configMap.Data = map[string]string{
+			"envoy.yaml": envoyConfig,
+		}
+		return nil
+	}); err != nil {
+		return err
+	}
+
+	if _, err := controllerutil.CreateOrUpdate(ctx, v.client, gateway, func() error {
+		gateway.Spec = v1beta1.Gateway{
+			Selector: map[string]string{
+				"istio": "ingressgateway",
+			},
+			Servers: []*v1beta1.Server{
+				{
+					Hosts: []string{*v.kubeAPIServerHost},
+					Port: &v1beta1.Port{
+						Name:     "tls-tunnel",
+						Number:   8132,
+						Protocol: "HTTP",
+					},
+				},
+			},
+		}
+		return nil
+	}); err != nil {
+		return err
+	}
+
+	if _, err := controllerutil.CreateOrUpdate(ctx, v.client, destinationRule, func() error {
+		destinationRule.Spec = v1beta1.DestinationRule{
+			ExportTo: []string{"*"},
+			Host:     fmt.Sprintf("%s.%s.svc.cluster.local", DeploymentName, v.namespace),
+			TrafficPolicy: &v1beta1.TrafficPolicy{
+				ConnectionPool: &v1beta1.ConnectionPoolSettings{
+					Tcp: &v1beta1.ConnectionPoolSettings_TCPSettings{
+						MaxConnections: 5000,
+						TcpKeepalive: &v1beta1.ConnectionPoolSettings_TCPSettings_TcpKeepalive{
+							Interval: &types.Duration{
+								Seconds: 75,
+							},
+							Time: &types.Duration{
+								Seconds: 7200,
+							},
+						},
+					},
+				},
+				Tls: &v1beta1.ClientTLSSettings{
+					Mode: v1beta1.ClientTLSSettings_DISABLE,
+				},
+			},
+		}
+		return nil
+	}); err != nil {
+		return err
+	}
+	if _, err := controllerutil.CreateOrUpdate(ctx, v.client, virtualService, func() error {
+		virtualService.Spec = v1beta1.VirtualService{
+			ExportTo: []string{"*"},
+			Hosts:    []string{*v.kubeAPIServerHost},
+			Gateways: []string{DeploymentName},
+			Http: []*v1beta1.HTTPRoute{
+				{
+					Route: []*v1beta1.HTTPRouteDestination{
+						{
+							Destination: &v1beta1.Destination{
+								Port: &v1beta1.PortSelector{
+									Number: 1194,
+								},
+								Host: DeploymentName,
+							},
+						},
+					},
+				},
+			},
+		}
+		return nil
+	}); err != nil {
+		return err
+	}
+
+	if _, err := controllerutil.CreateOrUpdate(ctx, v.client, service, func() error {
+		service.Annotations = map[string]string{
+			"networking.istio.io/exportTo": "*",
+		}
+		service.Spec.Type = corev1.ServiceTypeClusterIP
+		service.Spec.Ports = []corev1.ServicePort{
+			{
+				Name:       DeploymentName,
+				Port:       1194,
+				TargetPort: intstr.FromInt(1194),
+			},
+			{
+				Name:       "http-proxy",
+				Port:       9443,
+				TargetPort: intstr.FromInt(9443),
```

Comment:
Define a constant for `9443` and use it here and below in the envoy proxy config

Reply:
Done.